components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -385,20 +386,21 @@ function SpendingLimitReachedModal(p: { hints: any }) {
     return (
         <Modal visible={true} closeable={false} onClose={() => {}}>
             <h3 className="flex">
-                <span className="flex-grow">Spending Limit Reached</span>
+                <span className="flex-grow">Usage Limit Reached</span>
             </h3>
             <div className="border-t border-b border-gray-200 dark:border-gray-800 mt-4 -mx-6 px-6 py-2">
-                <p className="mt-1 mb-2 text-base dark:text-gray-400">Please increase the spending limit and retry.</p>
+                <Alert type="danger" className="app-container rounded-md">
+                    You have reached the <strong>spending limit</strong> of your billing account.
+                </Alert>
+                <p className="mt-1 mb-2 text-base dark:text-gray-400">
+                    Contant a team owner of <strong>{attributedTeam}</strong> to increase the spending limit, or change
```

Comment:
**fyi**: For now I just added the team name in bold here, in contrast with the designs proposed in https://github.com/gitpod-io/gitpod/issues/11927#issuecomment-1210813607 and https://github.com/gitpod-io/gitpod/issues/11927#issuecomment-1211812281.

Reply:
@gtsiolis why omit the link to the team? That link would be really useful for users.

Reply:
Typo 😇 

```suggestion
                    Contact a team owner of <strong>{attributedTeam}</strong> to increase the spending limit, or change
```

Reply:
Nice catch, @jankeromnes! ⚾ 
Will update in a moment.

Reply:
> @gtsiolis why omit the link to the team? That link would be really useful for users.

@jldec No specific reason, other than trying to avoid overloading the user with hyperlinks all over the place. We can also add this in a follow-up PR if we feel like it's missing. 💭

Reply:
Opened https://github.com/gitpod-io/gitpod/issues/12233 to keep track of this discussion. Resolving.